app/lib/frontend/templates/views/consent/publisher_member_invite.mustache

```diff
@@ -0,0 +1,19 @@
+{{! Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+<p>
+  <code>{{ active_account_email }}</code>
+  has invited you to be a member of the
+  <a href="https://dart.dev/tools/pub/verified-publishers" target="_blank" rel="noreferrer">verified publisher</a>
+  <a href="{{& publisher_url }}" target="_blank" rel="noreferrer"><code>{{ publisher_id }}</code></a>.
+</p>
+<p>
+  Accepting an publisher membership invitation means:
+</p>
+<ul>
+  <li>other members of the publisher will be able to see your email,</li>
+  <li>you will be able to publish new versions of packages owned by the publisher,</li>
+  <li>you will be able to transfer packages to be owned by the publisher,</li>
+  <li>you will be able to perform administrative actions for said publisher.</li>
```

Comment:
Instead of "the publisher" and "said publisher" let's just use the publisher name in `<code>` formatted text?